File: sentimental-cash/cash.py
from cs50 import get_float

# coin counter
coins = 0
#in a loot promnt user for change owed. Re-promnts if give invalid value
while True:
    coins_owed = get_float("Change owed: ")
    if coins_owed > 0:
        break

#rounds up coints
cent  = round(coins_owed * 100)


while cent >= 25:
    cent = cent - 25
    coins += 1


while cent >= 10:
    cent = cent - 10
    coins += 1


while cent >= 5:
    cent = cent - 5
    coins += 1


while cent >=1:
    cent = cent - 1
    coins += 1

#prints coins
print(coins)

File: migration-2/pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>


int main(void)

{
    
    // list of variables
       float dollars;
       int cents;
       int coins = 0;

    // Ask user how much change is owned to them
        
    do
       {
           
        dollars = get_float("How much change is owed?:");
        
       }
        
    // Keep repeating until user provides positive number
    while (dollars < 0);
        
    //round dollars to cents
        cents = round(dollars * 100);
        
        
    // To get minimum number of coins
    while (cents >= 25)
    {
        cents = cents - 25;
        coins++;
        
    }
        
    while (cents >= 10)
    {
        cents = cents - 10;
        coins++;
        
    }
    while (cents >= 5)
        
    {
            
        cents = cents - 5;
        coins++;
        
    }
    while (cents >= 1)
    {
            
        cents--;
        coins++;
        
    }
    //The number of coints owned to the user
    printf("%d\n", coins);
        
        
}
    